tools/c7n_mailer/README.md

```diff
@@ -120,13 +120,14 @@ policies:
           queue: https://sqs.us-east-1.amazonaws.com/1234567890/c7n-mailer-test
 ```
 
-There is a special `to` format that specifies datadog delivery, and includes the datadog configuration via url parameters.
-- metric_name: is the name of the metrics send to DataDog
-- metric_value_tag: by default the metric value send to DataDog is `1` but if you want to use one of the tags returned in the policy you can set it with the attribute `metric_value_tag`, for example in the `test-policy.yml` the value used is the size of the EBS volume. The value must be a number and it's transformed to a float value.
+There is a special `to` format that specifies Datadog delivery and it includes the Datadog configuration via URL parameters:
+
+- `metric_name`: is the name of the metrics send to DataDog.
+- `metric_value_tag`: by default the metric value to send to DataDog is `1`, but if you want to use one of the tags returned in the policy, you can set it with the attribute `metric_value_tag`. For example, in the `test-policy.yml` policy, the value used is the size of the EBS volume. The value must be a number and it will be transformed to a float value.
 
 ### Slack:
 
-The Custodian mailer supports Slack messaging as a separate notification mechanism for the SQS transport method. To enable Slack integration, you must specify a Slack token in the `slack_token` field under the `mailer.yml` file.
+The Custodian mailer supports Slack messaging as a separate notification mechanism for the SES transport method. To enable Slack integration, you must specify a Slack token in the `slack_token` field in your `mailer.yml` file. For example:
```

Comment:
This section is talking about notifying SQS and then targeting Slack at delivery time. SES isn't involved. The rest of the change looks good to me :+1: 

```suggestion
The Custodian mailer supports Slack messaging as a separate notification mechanism for the SQS transport method. To enable Slack integration, you must specify a Slack token in the `slack_token` field in your `mailer.yml` file. For example:
```